android/src/main/java/com/thebluealliance/androidclient/TBAAndroid.java

```diff
@@ -59,6 +59,9 @@ public TBAAndroid() {
     @Override
     public void onCreate() {
         super.onCreate();
+        if (LeakCanary.isInAnalyzerProcess(this)) return;
```

Comment:
Thanks for the fix!

I recommend using braces around each "then" and "else" branch except maybe some one-liners like this line. Even when formatting on one line, I usually write explicit braces. Otherwise it's easy to accidentally break code e.g. by adding a Log.d() call.

Shall we copy in the explanatory comment from [LeakCanary README](https://github.com/square/leakcanary)?
```
      // This process is dedicated to LeakCanary for heap analysis.
      // You should not init your app in this process.
```

After then-return, `else` is confusing; the `LeakCanary.install(this)` call is effectively in the same block as the following code.